test/integration/supervisor_federationdomain_status_test.go

```diff
@@ -678,6 +702,7 @@ func TestSupervisorFederationDomainCRDValidations_Parallel(t *testing.T) {
 			wantKube23OrOlderErrs:            []string{`spec.identityProviders.transforms.constants.name: Invalid value: "12345678901234567890123456789012345678901234567890123456789012345": spec.identityProviders.transforms.constants.name in body should be at most 64 chars long`},
 			wantKube24Through31InclusiveErrs: []string{`spec.identityProviders[0].transforms.constants[0].name: Too long: may not be longer than 64`},
 			wantKube32OrNewerErrs:            []string{`spec.identityProviders[0].transforms.constants[0].name: Too long: may not be more than 64 bytes`},
+			wantCELErrs:                      []string{`<nil>: Invalid value: "null": some validation rules were not checked because the object was invalid; correct the existing errors to complete validation`},
```

Comment:
This error message seems wrong for the test case, which is about the length of constants. Same applies to a couple of other test cases below. Perhaps adjust the spec inputs to avoid this?

Reply:
I believe this vague error message is shown because the `kubebuilder:validation:MaxLength` is checked before any CEL validations are run. If the `MaxLength` check fails, the CEL validations are not run. This is just the API server making sure you know that there are "other validations" that weren't run so it has no way of knowing whether those other validations would fail if you got past the `MaxLength` check.

Reply:
Strange behavior, but I suppose that's out of our control. Thanks for looking into it.